### PR TITLE
Include width and height from addWindow action

### PR DESCRIPTION
**Pull request by SaravShah**
completed during code club with Jeremy, Jessie, Joe, Camille, Jack and Drew

### CODE REVIEW

File: minimal_redux_poc/src/components/Window.js [hunk 1]
```diff
@@ -15,13 +15,21 @@ class Window extends Component {
     return thumb.id;
   }
 
+  /**
+   * Get width and height
+   */
+  stylnProfiln() {
```
Comment (camillevilla) [inline minimal_redux_poc/src/components/Window.js]: 😎

**Comment by mejackreed** [comment]
@SaravShah we would love to get this in. Are you planning on adding some tests, if not would you mind if we did?

**Comment by SaravShah** [comment]
@mejackreed happy for you to take it over, also happy to work on it during code club Wednesday. Whatever works!

**Pull request merged**